src/compiler/binder.ts

```diff
@@ -123,7 +123,7 @@ module ts {
                 case SyntaxKind.ExportDeclaration:
                     return "__export";
                 case SyntaxKind.ExportAssignment:
-                    return "default";
+                    return (<ExportAssignment>node).isExportEquals ? "export=" : "default";

```

Comment:
this is a fairly odd name to give this entity.  Is it used elsewhere?  if so, we should put it in a common location so we don't have magic strings like this floating around.